### DISCUSSION TITLE
Use `jom` to build OpenSSL on Windows

### DISCUSSION THREAD

**Discussion by Nerixyz** [conan-io/conan-center-index#26263] [Maintenance]
The OpenSSL builds on Windows are really slow because they use `nmake` which can't parallelize the OpenSSL build (especially when updating the OpenSSL version in CI, this takes quite long). Qt's `jom` can parallelize the build (see also: https://github.com/openssl/openssl/issues/9931). It can run the `Makefile` generated by OpenSSL on Windows.

Vcpkg also uses `jom` to build on Windows
- OpenSSL portfile: https://github.com/microsoft/vcpkg/blob/80d54ff62d528339c626a6fbc3489a7f25956ade/ports/openssl/windows/portfile.cmake#L76 
- vcpkg_build_nmake: https://github.com/microsoft/vcpkg/blob/80d54ff62d528339c626a6fbc3489a7f25956ade/scripts/cmake/vcpkg_build_nmake.cmake#L55-L63
- vcpkg_find_acquire_program(JOM): https://github.com/microsoft/vcpkg/blob/80d54ff62d528339c626a6fbc3489a7f25956ade/scripts/cmake/vcpkg_find_acquire_program(JOM).cmake

**Comment by jcar87** [comment]
Continued in https://github.com/conan-io/conan-center-index/issues/26349
